deap/tools/crossover.py

```diff
@@ -428,17 +443,62 @@ def cxESTwoPoint(ind1, ind2):
 
     return ind1, ind2
 
+
 def cxESTwoPoints(ind1, ind2):
     """
     .. deprecated:: 1.0
         The function has been renamed. Use :func:`cxESTwoPoint` instead.
     """
     return cxESTwoPoint(ind1, ind2)
 
+
+######################################
+# ES Crossovers                      #
+######################################
+
+def semantic_crossover(ind1, ind2, gen_func=gp.genGrow, pset=None, min=2, max=6):
+    """
+    Implementation of the Semantic Crossover operator
```

Comment:
A little description of what it does should be included here, an example would be a nice to have, and a reference to the paper is mandatory.